tests/unit/test_charm.py

```diff
@@ -309,3 +309,65 @@ def test_minio_console_port_args(harness):
         "--console-address",
         ":9999",
     ]
+
+def test_install_with_all_inputs(harness):
```

Comment:
This test can work as a general "can I successfully invoke pod_spec" test, but I think we are missing something to actually check whether this charm has published what we expect to the relation.  For example, like [here](https://github.com/canonical/kfp-operators/blob/d974ec1426a61729daf2718484ff503773d5bb06/charms/kfp-ui/tests/unit/test_operator.py#L292)